staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1369,29 +1369,44 @@ type Condition struct {
 	// ---
 	// Many .condition.type values are consistent across resources like Available, but because arbitrary conditions can be
 	// useful (see .node.status.conditions), the ability to deconflict is important.
+	// The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
 	// +required
+	// +kubebuilder:validation:Required
```

Comment:
it's not perfect, but the max length is 253+63=316

Reply:
added